addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/discovery/HarmonyHubDiscovery.java

```diff
@@ -52,7 +52,7 @@
 
     private static final int DISCO_PORT = 5224;
 
-    static protected final ScheduledExecutorService scheduler = ThreadPoolManager
+    private static final ScheduledExecutorService SCHEDULER = ThreadPoolManager
```

Comment:
since the class does not inherit from something with an existing scheduler I believe I chose to use org.eclipse.smarthome.core.common.ThreadPoolManager instead (its been a while, so I don't exactly remember).